tests/workflow.go

```diff
@@ -980,3 +983,74 @@ func (s *FunctionalSuite) TestWorkflowRetryFailures() {
   4 WorkflowTaskCompleted
   5 WorkflowExecutionFailed`, events)
 }
+
+func (s *FunctionalSuite) TestExecuteMultiOperation() {
+	s.Run("Start Workflow and send Update", func() {
```

Comment:
Thank you for describing what scenario we're testing. We need to do that more often